pkg/wsman/amt/ethernetport/settings_test.go

```diff
@@ -122,7 +122,42 @@ func TestAMT_EthernetPortSettings(t *testing.T) {
 				},
 			},
 			//PULLS
-			// {"should create a valid AMT_EthernetPortSettings Pull wsman message", "AMT_EthernetPortSettings", wsmantesting.PULL, wsmantesting.PULL_BODY, func() string { return elementUnderTest.Pull(wsmantesting.EnumerationContext) }},
+			{
+				"should create a valid AMT_EthernetPortSettings Pull wsman message",
+				"AMT_EthernetPortSettings",
+				wsmantesting.PULL,
+				wsmantesting.PULL_BODY,
+				"",
+				func() (Response, error) {
+					currentMessage = "Pull"
+					return elementUnderTest.Pull(wsmantesting.EnumerationContext)
+				},
+				Body{
+					XMLName: xml.Name{Space: "http://www.w3.org/2003/05/soap-envelope", Local: "Body"},
+					PullResponse: PullResponse{
+						Items: []Item{
+							{
+								EthernetPort: EthernetPort{
+									DHCPEnabled:            true,
```

Comment:
@zaidusmani26 can you also add "IPAddress: string," to this struct?  Seems that got missed when originally adding these properties